samples/operate-first/requirements.txt

```diff
@@ -0,0 +1 @@
+pyyaml
```

Comment:
Please fixate it to version `5.4.1` because earlier versions had a security issue (we just dealt with that in the flight module project)

Reply:
Done!